docs/en/advanced_guides/datasets/nuscenes.md

```diff
@@ -184,6 +185,40 @@ It follows the general pipeline of 2D detection while differs in some details:
 - Some data augmentation techniques need to be adjusted, such as `RandomFlip3D`.
   Currently we do not support more augmentation methods, because how to transfer and apply other techniques is still under explored.
 
+2. BEV-based
+   BEV, Bird's-Eye-View, is another popular 3D detection paradigm. It directly takes multi-view images to perform 3D detection, for nuScenes, they are `CAM_FRONT`, `CAM_FRONT_LEFT`, `CAM_FRONT_RIGHT`, `CAM_BACK`, `CAM_BACK_LEFT` and `CAM_BACK_RIGHT`. A basic training pipeline of bev-based 3D detection on nuScenes is as below.
+
+```python
+    train_pipeline = [
+        dict(type='LoadMultiViewImageFromFiles',
+             to_float32=True, num_views=6),
+        dict(type='LoadAnnotations3D', ),
+        dict(type='Pack3DDetInputs',
```

Comment:
Better to add some common `aug` usage for BEV-based detection, such as `Resize3D`